python/samples/getting_started_with_agents/multi_agent_orchestration/step3_group_chat.py

```diff
@@ -0,0 +1,117 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+
+from semantic_kernel.agents import Agent, ChatCompletionAgent, GroupChatOrchestration
+from semantic_kernel.agents.orchestration.group_chat import RoundRobinGroupChatManager
```

Comment:
Can we simplify imports? For example, in main we have: `from semantic_kernel.agents import Agent, ChatCompletionAgent, ConcurrentOrchestration`

Reply:
Indeed, since you need one of the GroupChatManager classes (or a custom one based on the base) they should be part of Agents as well